dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/maatapi/model/fdc/FdcContributionEntry.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.justice.laa.crime.dces.integration.maatapi.model.fdc;
```

Comment:
I guess you haven't moved this, but it's odd that some model classes are in `.../integration/model/**` packages, whereas others are in `.../integration/maatapi/model/**` packages. I don't see any logic to it.